pkgs/by-name/ms/msedgedriver/package.nix

```diff
@@ -0,0 +1,84 @@
+{
+  autoPatchelfHook,
+  fetchzip,
+  glib,
+  lib,
+  libxcb,
+  nspr,
+  nss,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "msedgedriver";
+  # finding a version that has all 4 builds is a pain
+  # https://msedgewebdriverstorage.z22.web.core.windows.net/?form=MA13LH
+  version = "130.0.2849.1";
```

Comment:
An update script would be nice, but this issue makes it harder to do that.

Reply:
Do we have an API to query versions of MS EDGE perhaps?